docs/datascience/data-wrangler-quick-start.md

```diff
@@ -0,0 +1,87 @@
+# Quick Start Guide for Data Wrangler in VS Code
```

Comment:
Make sure to add file metadata.
Also include the new article in the feedback system (Nick).